tests/images.rs

```diff
@@ -380,3 +381,56 @@ fn orientdb_exists_database() {
 
     assert!(!exists);
 }
+
+#[test]
+fn kafka_publish_and_read_message() {
+    let free_local_port = free_local_port().unwrap();
+    let docker = clients::Cli::default();
+    let zk_image =
+        images::zookeeper::Zookeeper::default().with_mapped_port((free_local_port, 2181));
```

Comment:
I don't know much about zookeeper and kafka but if these images are supposed to talk to each other, you might want to utilize the functionality for docker networks that was added recently: https://github.com/testcontainers/testcontainers-rs/pull/206